src/helper/findFailedTestDetails.tsx

```diff
@@ -83,3 +84,11 @@ export const findTestDetailsForEnv = (environments: Environments, envName: strin
   }
   return createDataObject(envName, 'pass', pass);
 };
+
+const findVrtData = (environments: Record<string, any>): FailedTest => {
+  const failedTest = {} as FailedTest;
+  const name = Object.keys(environments)[0];
```

Comment:
this will fail in multiple environments

Reply:
We will have only one environment data in VRT (defaults) since we are flattening all env data.